webapp/components/PostCard/PostCard.vue

```diff
@@ -117,6 +118,10 @@ export default {
         this.deletePostCallback,
       )
     },
+    language() {
+      // if older posts have language not set I assume German as language
+      return this.post.language ? this.post.language : 'de'
+    },
```

Comment:
I don't feel comfortable to have a default set to German *in the code base*. If, at all, I would suggest to use English as default. But in fact my suggestion is to remove this default so our users have an incentive to fix their old posts.

Reply:
I agree, that it is no good idea to have this in the code base. I did this as a pragmatic solution for the 18k posts without language property, assuming that they are mainly written in German. New posts should have set the language property always, so the default here does not apply.